prescriptions-refresh-job/overlays/ocp4-stage/imagestreamtag.yaml

```
@@ -0,0 +1,14 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: prescriptions-refresh-job
+spec:
+  tags:
+    - name: latest
+      from:
+        kind: DockerImage
+        name: quay.io/thoth-station/prescriptions-refresh-job:kebechet-v0.5.0
```

```suggestion
        name: quay.io/thoth-station/prescriptions-refresh-job:v0.5.0
```
This image is now available